app/controllers/application_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ApplicationController < ActionController::Base
+  include ApplicationHelper
+  def signout
+    @current_user = nil
+    session[:user_id] = nil
```

Comment:
Session params are key value pairs stored in a hash. If you are setting the a `key` to `nil` you're only deleting the  `user_id` value, but leaving the `key` in there. If you had a `session = {:user_id: 1}` hash, you'll now have a `session = {:user_id: nil}`. What you really want to do is to `delete` the key and its value altogether. And we can do this with the `delete` method [here](https://apidock.com/rails/ActionDispatch/Request/Session/delete):
```
session.delete(:user_id)
```

[**REQUIRED**]